lib_nbgl/doc/nbgl_use_case.dox

```diff
@@ -653,122 +326,107 @@ static nbgl_layoutTagValue_t* getPair(uint8_t index) {
+@subsection use_case_review_light Light Static Review Use Case
 
 In some cases, the developer may want to display a review but with a less intense confirmation than a long-press button. A simple button is used in this
 case.
 
-The API to initiate the display of the series of pages is @ref nbgl_useCaseStaticReviewLight(), providing:
+The API to initiate the display of the series of pages is @ref nbgl_useCaseReviewLight(), providing:
 
 - the list of tag/value pairs (or a callback to get them one by one)
-- the texts to use in footer and in last page (icon, title, confirmation button text)
-- a callback called when the black button on last page or the footer to reject is used. The callback's param is true for confirmation, false for rejection.
+- the texts/icon to use in presentation page and in last page
+- a callback called when the long press button on last page or reject confirmation is used. The callback's param is *true* for confirmation, *false* for rejection.
+
+@subsection use_case_review_streaming Streaming review Use Case
+
+\image{inline} html UseCase-Streaming.png "caption" height=300
+
+In some cases, the application cannot know all tag/value pairs of a transaction when the review is started.
+
+In this case, what we call a "streaming" review can be used. The pages to display for each "stream are computed automatically and pages can be navigated forward
+and backward (only within a "stream" for backward).
+
+In case of a tag/value pair too long to be fully displayed, the "more" button will be automatically drawn and its handling
+automatically performed by NBGL by building a detailed modal view.
 
-@subsection use_case_details_view Details View Use Case
+When the user taps on "Reject" in any page, a confirmation page is automatically drawned to let user confirm that he rejects
+the transaction. In this case, the given callback is called and it's up to app's developer to call @ref nbgl_useCaseReviewStatus(), as in
+case of long-press.
 
-\image{inline} html UseCase-DetailsView.png "caption" height=300
+The API to initiate the display of the series of pages is @ref nbgl_useCaseReviewStreamingStart(), providing:
 
-Sometimes some values in tag/value pairs are too long to be displayed in a transaction/message review, and for that a
-special content type (@ref TAG_VALUE_DETAILS) to display only the first lines of value, ended with "...".
-And a button (for example with "More" text) is display to offer to the end-user a way to view details on this
-tag/value, in a modal set of pages.
+- the type of operation to review (transaction, message or generic operation)
+- the texts/icon to use in presentation page
+- a callback with one boolean parameter.
+  - If this parameter is *false*, it means that the transaction is rejected.
+  - If this parameter is *true*, it means that NBGL is waiting for new data, sent with @ref nbgl_useCaseReviewStreamingContinue() or @ref nbgl_useCaseReviewStreamingContinueExt()
```

Comment:
the APi to call is already (better) explained just below. I suggest to remove the end of the line, after `new data`?

Reply:
I will reword it